model/enduser/registry.yaml

```diff
@@ -0,0 +1,32 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes information about the end user, which can be used as a subdomain of browser, client, or user domains.
+    attributes:
+      - id: enduser.id
+        type: string
+        stability: experimental
+        brief: >
+          Identifier of an end user who interacts with a system.
+          This identifier may be unique only through best-effort means and does not imply that the user is authenticated to the system.
+        examples: ['QdH5CAWJgqVT4rOr0qtumf']
+      - id: enduser.role
+        type: string
+        stability: experimental
+        brief: 'Actual/assumed role the client is making the request under extracted from token or application security context.'
+        examples: 'admin'
+      - id: enduser.scope
+        type: string
+        stability: experimental
+        brief: >
+          Scopes or granted authorities the client currently possesses extracted from token
+          or application security context. The value would come from the scope associated
+          with an [OAuth 2.0 Access Token](https://tools.ietf.org/html/rfc6749#section-3.3)
+          or an attribute value in a [SAML 2.0 Assertion](http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html).
+        examples: 'read:message, write:files'
+      - ref: authentication.id
```

Comment:
@trisch-me need help with this. 

I have tried a few options, none works so far

```yaml
- id: enduser.authentication
- ref: registry.authentication
```
or 
```yaml
-id: enduser.authentication
-type:
    members:
        - ref: authenticaiton.id
```
current yaml allows me to update markdown but it breaks policy check. any ideas?

Reply:
You would have:

```
- id: enduser.authentication.id
  ...
```

You do not need to make the `authenticaton` namespace if you want it under `enduser`